command/commands/user.py

```diff
@@ -121,6 +123,7 @@ def handle(self, command, user_id):
                 "github": args.github,
                 "major": args.major,
                 "bio": args.bio,
+                "perm": args.permission,
```

Comment:
I think this should be changed to be `permission` just so that we are consistent with all the other arguments and their associated parameter. 
```
        parser_edit.add_argument("--name", type=str, action='store')
        parser_edit.add_argument("--email", type=str, action='store')
        parser_edit.add_argument("--pos", type=str, action='store')
        parser_edit.add_argument("--github", type=str, action='store')
        parser_edit.add_argument("--major", type=str, action='store')
        parser_edit.add_argument("--bio", type=str, action='store')
        parser_edit.add_argument("--member", type=str, action='store')
        parser_edit.add_argument("--permission", type=lambda x: Permissions[x],
                                 action='store', choices=list(Permissions))
```
and
```
            param_list = {
                "member": args.member,
                "name": args.name,
                "email": args.email,
                "pos": args.pos,
                "github": args.github,
                "major": args.major,
                "bio": args.bio,
                "perm": args.permission,
            }
```